apps/src/lib/bench_utils.rs

```diff
@@ -111,6 +111,10 @@ pub const TX_WITHDRAW_WASM: &str = "tx_withdraw.wasm";
 pub const TX_INIT_ACCOUNT_WASM: &str = "tx_init_account.wasm";
 pub const TX_INIT_VALIDATOR_WASM: &str = "tx_init_validator.wasm";
 
+pub const TX_RESIGN_STEWARD: &str = "tx_resign_steward.wasm";
```

Comment:
how do these changes relate to this PR

Reply:
Ah, we were just missing the benchmark for this specific transaction, so I added it in this PR, nothing more than this